app/services/ClaimToAdjustHelper.scala

```diff
@@ -134,4 +141,39 @@ trait ClaimToAdjustHelper {
       .sortBy(_.taxYear).reverse.headOption.map(doc => makeTaxYearWithEndYear(doc.taxYear))
   }
 
+  def getAmendablePoaViewModel(documentDetails: List[DocumentDetail],
```

Comment:
its the same method name as in actual ClaimToAdjustService but with a different set of parameters, maybe we can change method name to createAmendablePoaViewModel ? or similar to have to differentiate between these two;